src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -783,6 +783,20 @@ public function prepare_item_for_response( $item, $request ) {
 					}
 
 					$size_data['source_url'] = $image_src[0];
+
+					if ( empty( $data['media_details']['sizes'][ $size ]['sources'] ) || ! is_array( $data['media_details']['sizes'][ $size ]['sources'] ) ) {
+						continue;
+					}
+
+					$size_data_sources = array();
+					$directory = dirname( $data['media_details']['sizes'][ $size ]['source_url'] );
+					foreach ( $data['media_details']['sizes'][ $size ]['sources'] as $mime => $mime_details ) {
+						$source_url                 = "{$directory}/{$mime_details['file']}";
+						$mime_details['source_url'] = $source_url;
+						$size_data_sources[ $mime ] = $mime_details;
```

Comment:
This line here won't be needed if we make `&$mime_details` accessible as above.

Reply:
Thank you for the review and for pointing it out. This feedback is addressed.